src/index.html

```diff
@@ -13,6 +13,38 @@
     />
```

Comment:
Link to SCSS file is incorrect. You should link to the compiled CSS file instead, which is usually named `styles.css` or `index.css`. SCSS files are preprocessed on the server side and browsers do not understand SCSS syntax.